content/developers/_index.md

```diff
@@ -1,48 +1,37 @@
 ---
 title: Developer Tools
 kind: documentation
-description: Collect custom metrics & create new integration
+description: Everything you need to know about developing for the Datadog platform, including config and code examples, and a ton of reference material.
 ---
 
-Monitoring tools should be flexible, so we provide multiple ways to interact with Datadog. This section gives you a quick overview of the various methods available
+This section includes technical details, code examples, and reference documentation to help get you up and running quickly.
 
-## API
-Datadog features a powerful API to programmatically access the service. You can submit custom metrics; manage dashboards and monitors; schedule downtimes; and perform administrative actions.
-Available endpoints can be found on [our dedicated API documentation page][1].
+#### Custom Checks
 
-In addition, [many libraries][2] are available to directly interact with the API.
+Wondering how to report your own custom Checks? You've got options; start with the sections on [Agent Checks][1] and [Prometheus Checks][2], which contain tons of configuration and code examples. From there, move on to the [DogStatsD section][3] to learn how to use the metrics aggregation service built in to the Agent itself.
```

Comment:
r/`You've got`/`You have got`

Reply:
I don't even know if it's necessary. We could just say:

`Wondering how to report your own custom Checks? Start with the sections on ...`